<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_signup">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>721</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Elephant</family>
     <pointsize>14</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="title">
    <string>SIGN UP</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>701</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_email">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>EMAIL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_username">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>USERNAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_password">
         <property name="font">
          <font>
           <family>Bahnschrift</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>PASSWORD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="LineEdit_email"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_username"/>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_password">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MAIN MENU</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
